multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -51,14 +51,34 @@ def multilingual():
 
 
 @automm_presets.register()
-def zero_shot():
+def zero_shot_classification():
     return {
+        "model.names": ["hf_text"],
+        "model.hf_text.checkpoint_name": "cross-encoder/ms-marco-MiniLM-L-12-v2",
+        "env.eval_batch_size_ratio": 1,
+    }
+
+
+@automm_presets.register()
+def zero_shot_image_classification():
+    return {
+        "model.names": ["clip"],
         "model.clip.checkpoint_name": "openai/clip-vit-large-patch14-336",
         "model.clip.max_text_len": 0,
         "env.eval_batch_size_ratio": 1,
     }
 
 
+@automm_presets.register()
+def feature_extraction():
```

Comment:
Ditto. Is `feature_extraction` only for text?

Reply:
It's should be able to apply to any models.

Reply:
But setting `"model.names": ["hf_text"]` only allows using the hf text models.

Reply:
Yes, this preset is for text only. Or do we need to change it to clip to support both text and image?